tests/ui/consts/const-mut-refs/mut_ref_in_final.stderr

```diff
@@ -4,8 +4,14 @@ error[E0764]: mutable references are not allowed in the final value of constants
 LL | const B: *mut i32 = &mut 4;
    |                     ^^^^^^
 
+error: encountered mutable pointer in final value of constant
+  --> $DIR/mut_ref_in_final.rs:10:1
+   |
+LL | const B: *mut i32 = &mut 4;
+   | ^^^^^^^^^^^^^^^^^
```

Comment:
This error is basically a duplicate of the one above. Why do we get both of them now?

Reply:
One is  const checking failure, the other is an evaluation failure. I'll try to make const eval not run if checking failed